python_modules/libraries/dagster-dbt/dagster_dbt_tests/core/test_asset_checks.py

```diff
@@ -76,7 +76,13 @@ def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
     )
     assert my_dbt_assets.check_specs_by_output_name
 
-    assert my_dbt_assets.check_specs_by_output_name == {
+    # too annoying to manually input all of the individual metadata fields
+    stripped_specs_by_output_name = {}
+    for output_name, spec in my_dbt_assets.check_specs_by_output_name.items():
+        assert spec.metadata and spec.metadata[DAGSTER_DBT_UNIQUE_ID_METADATA_KEY]
```

Comment:
use check.invariant; asserts in app code are bad practice - they are removed when running in non-debug mode.

Reply:
this is a test brother